app/controllers/cats_controller.rb

```diff
@@ -5,6 +5,8 @@ def index
 end
 
 def create
+    cats =  Cat.create(cat_params)
```

Comment:
you are only creating a single cat so the variable style guide would require that this be "cat" not "cats"